#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_check_log_fix_basename_path.dpatch by Jan Wagner <waja@cyconet.org>
##
## DP: Drop absolut patch from basename

From d4b6d09f104f73c462e0e0c8a297ba9fc7c5a53e Mon Sep 17 00:00:00 2001
From: Jan Wagner <waja@cyconet.org>
Date: Wed, 20 Aug 2014 20:29:23 +0200
Subject: [PATCH] check_log.sh: droping path from basename while evaluating
 PROGNAME (Closes Debian Bug #758662)

Origin: upstream, https://github.com/monitoring-plugins/monitoring-plugins/commit/d4b6d09f104f73c462e0e0c8a297ba9fc7c5a53e.patch
Bug: https://bugs.debian.org/758662
---
 plugins-scripts/check_log.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

@DPATCH@
diff --git a/plugins-scripts/check_log.sh b/plugins-scripts/check_log.sh
index 8653a5e..f67eff6 100755
--- a/plugins-scripts/check_log.sh
+++ b/plugins-scripts/check_log.sh
@@ -58,7 +58,7 @@
 # Paths to commands used in this script.  These
 # may have to be modified to match your system setup.
 
-PROGNAME=`/bin/basename $0`
+PROGNAME=`basename $0`
 PROGPATH=`echo $0 | sed -e 's,[\\/][^\\/][^\\/]*$,,'`
 REVISION="@NP_VERSION@"
 PATH="@TRUSTED_PATH@"
-- 
2.0.4

